mix.exs

```diff
@@ -4,13 +4,13 @@ defmodule Kaffe.Mixfile do
   def project do
     [
       app: :kaffe,
-      version: "1.17.0",
+      version: "1.18.0",
       description:
         "An opinionated Elixir wrapper around brod, the Erlang Kafka client, that supports encrypted connections to Heroku Kafka out of the box.",
       name: "Kaffe",
       source_url: "https://github.com/spreedly/kaffe",
       package: package(),
-      elixir: "~> 1.3",
+      elixir: "~> 1.7",
```

Comment:
What's the semver implication of this? Does this imply needing to bump a major version?

Reply:
Not sure we need a major bump. A minor bump will still allow older runtimes to pin to `kaffe ~> 1.17.0`.

Reply:
@objectuser Good question! At first I agreed, but then read [this article](https://medium.com/@sampart/semantic-versioning-when-you-change-the-required-programming-language-version-16a3a3555c95), which seems to fit what we're doing here. They end up settling for bumping the minor version. A penny for your thoughts 💰

Reply:
I suppose it's a reasonable argument. I don't have a better one. 👍